ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -1556,7 +1557,14 @@ private static ConfigDef buildConfigDef(final ConfigGeneration generation) {
             Importance.LOW,
             KSQL_FETCH_REMOTE_HOSTS_TIMEOUT_SECONDS_DOC
         )
-        .withClientSslSupport();
+        .withClientSslSupport()
+        .define(
+            ConfluentConfigs.ENABLE_FIPS_CONFIG,
+            Type.BOOLEAN,
+            ConfluentConfigs.ENABLE_FIPS_DEFAULT,
+            Importance.LOW,
```

Comment:
whats the meaning of this low importance?

Reply:
+1

Reply:
The importance of a config gives the user a feeling how to prioritize the tuning of the config. This concept of the importance of a config comes from AK. For example in Kafka Streams the following configs have a high importance:
- `application.id`: This config is required without defaults. It influences the names of internal topics, the name of the consumer group and other fundamental names.
- `bootstrap.servers`: This config is required without defaults. It represents the host for the initial connection to Kafka brokers. Without this config or with the wrong value the Streams application does not work.
-  `state.dir`: This config represents the local folder in which the state is stored. By default, it is the temp directory. The default value is good for getting started with Streams but is bad for production because the temp directory might get deleted inadvertently.
In general, those configs should be the first to look at.

Since FIPS seems to be a specific case, I would say `Importance.LOW` is appropriate but you know it probably better.